module-apps/application-phonebook/windows/PhonebookNewContact.cpp

```diff
@@ -73,6 +73,10 @@ namespace gui
         }
 
         !newContactModel->emptyData() ? setSaveButtonVisible(true) : setSaveButtonVisible(false);
+
+        if (auto message = dynamic_cast<PhonebookItemData *>(data); message != nullptr) {
```

Comment:
wishful thinking - there is no model to first check it -> then operate the UI to use?

Reply:
done